cypress/e2e/shared/arduino.test.ts

```diff
@@ -1,10 +1,13 @@
 describe('Arduino onboarding flow', () => {
+  const isIOxOrg = Boolean(Cypress.env('useIox'))
+
   beforeEach(() => {
     cy.flush()
```

Comment:
My bad that I didn't catch it eariler. I think this line should be in `beforeEach()` block so that each `it()` and `describe()` will be skipped on IOx
```
cy.skipOn(isIOxOrg)
```